.github/workflows/project_issues.yml

```diff
@@ -1,18 +1,83 @@
-name: Project Automation - move issues
+name: Project Atuomation - Move Issues
 
 on:
+  pull_request:
+    types: [opened, synchronize, reopened]
   issues:
     types: [labeled]
 
+env:
+  PAT_TOKEN: ${{ secrets.PAT_TOKEN }}
```

Comment:
Is this naughty - it can only work with a PAT. I've used one of mine.

@elglup Can you generate one via the bot account for me? It needs the following scopes:

>repo (Full control of private repositories)
>admin:org (Full control of orgs and teams, read and write org projects)